docs/versioning.md

```diff
@@ -0,0 +1,48 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+
+# Versioning
+
+`tezos-packaging` provides support for all Octez stable releases and RC candidates.
+
+However, in some cases we provide multiple releases within the same upstream version
+because our packages provide additional functionality (e.g. systemd services in Ubuntu
+and Fedora packages or brew formulae with launchd services) that may need to be updated
+within the single upstream release.
+
+In order to achieve that, our GitHub releases and packages use the following RPM-like versioning scheme:
+`<upstream-version>-<release-number>`, e.g. `v11.0-1` for the first `tezos-packaging` release
+within the `v11.0` upstream stable release, or `v11.0-rc2-2` for the second `tezos-packaging` release
+within the `v11.0-rc2` upstream release candidate.
```

Comment:
from what i read only RPM versioning is more like:
> [name]-[version]-[release]

all of these 3 values apply to our versioning scheme too. So I suggest:
-  to mention that this scheme is applied to binaries too, which have a `name`
- I would use `version` more in the explanations to indicate the upstream releases